samples/crypto/test_sample/README.rst

```diff
@@ -0,0 +1,258 @@
+.. _crypto_test:
+
+Cryptography test sample
+########################
+
+The cryptography test sample verifies the functionality of the nrf_crypto operations by using known test vectors approved by the National Institute of Standards and Technology (NIST) and others.
+All available nrf_crypto backends can be tested by modifying the nrf_crypto section in the sdk_config file, as described in lib_crypto_config.
```

Comment:
Ensure all references to "nrf_crypto" is changed to state "nrf_security" e.g. nrf_security backends

Reply:
This has been rewritten. I see this is handled in a subsequent issue in the PR